samples/certs/README.md

```diff
@@ -14,6 +14,10 @@ The included sample files are:
 - `cert-chain-alt.pem`: alternative certificate chain.
 - `workload-foo-[cert|key].pem`: workload certificate and key for URI SAN `spiffe://trust-domain-foo/ns/foo/sa/foo` signed by `ca-cert.key`.
 - `workload-bar-[cert|key].pem`: workload certificate and key for URI SAN `spiffe://trust-domain-bar/ns/bar/sa/bar` signed by `ca-cert.key`.
+- `workload-foo-root-certs.pem`: root and intermediate CA certificates for foo workload certificate.
+- `workload-bar-root-certs.pem`: root and intermediate CA certificates for bar workload certificate.
+- `leaf-workload-foo-cert.pem`: leaf workload certificate
+- `leaf-workload-bar-cert.pem`: leaf workload certificate
```

Comment:
Can we add a period to the end of the sentences. Also we mention foo and bar in the above sentences but not these.

Reply:
Updated!